### PR TITLE
Basic analysis env

### PR DESCRIPTION
**Pull request by jthorton**
## Description
For users doing the submissions or final analysis from mac or a different PC this env will build the minium environment needed to do this. @dfhahn Please could advise what packages are needed only for the analysis steps? 

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [X] Strip out all compute engines (Psi4)
 - [x] Check what packages are needed for analysis
 - [ ] Check that preprocessing still works in this env
 - [ ] Check that task submission still works in this env

## Status
- [ ] Ready to go

### CODE REVIEW

File: devtools/prod-envs/openff-benchmark-analysis.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  # QCPortal dependencies
+  - double-conversion >=3.0.0
+  - h5py
+  - pandas
+  - plotly >=4.0.0
+  - pyarrow >=0.13.0
+  - tqdm
+
+  # MM calculations
+  - openforcefield =0.8.0
```
Comment (j-wags) [inline devtools/prod-envs/openff-benchmark-analysis.yaml:28]: Let's bump this up to `0.8.2` to get the stereochemistry fixes
```suggestion
  - openforcefield =0.8.2
```

**Comment by dotsdl** [comment]
@dfhahn can you have a look at the software you need for analysis and add it to this env?

**Comment by dfhahn** [comment]
@jthorton these packages are needed:
numpy
pandas
openforcefield
rdkit
pint
itertools
matplotlib
seaborn
plotly

**Review by j-wags — APPROVED** [review]
LGTM!

**Comment by dotsdl** [comment]
@jthorton whenever ready, feel free to merge!

**Pull request merged**